website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md

```diff
@@ -38,62 +40,40 @@ To improve your experience using dbt Cloud, we suggest that you turn off ad bloc
 - A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
 - A git repository set up and git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/cloud/git/connect-github) or [Importing a project by git URL](/docs/cloud/git/import-a-project-by-git-url) for detailed setup instructions
 - A dbt project connected to a [data platform](/docs/cloud/connect-data-platform/about-connections)
-- A [development environment and development credentials](#access-the-cloud-ide) set up
+- A [development environment and development credentials](#get-started) set up
 - The environment must be on dbt version 1.0 or higher
 
-
 ## dbt Cloud IDE features
 
 The dbt Cloud IDE comes with features that make it easier for you to develop, build, compile, run, and test data models. 
 
 To understand how to navigate the IDE and its user interface elements, refer to the [IDE user interface](/docs/cloud/dbt-cloud-ide/ide-user-interface) page.
 
-
-| Feature  |  Info |
+| Feature  |  Description |
```

Comment:
Just a note for later - it might be better to make this table more concise and then give each feature it's own instructional section further down in the doc, rather than listing instructional material in a table